operator/internal/manifests/rules_config.go

```diff
@@ -8,41 +8,54 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+type RuleName struct {
+	cmName      string
+	tenantID    string
+	ns_name_uid string
```

Comment:
Go naming demands camel-case, anyway this field represents the filename after projecting the configmaps, so why not calling it `filename`?